rllib/models/tf/visionnet.py

```diff
@@ -1,3 +1,5 @@
+import numpy as np
```

Comment:
The changes here make sure that Conv2D behave exactly the same as FullyConnected wrt their wrapping behavior.